modules/whosOnFirst.py

```diff
@@ -77,20 +79,22 @@ def get_svg(self, led):
             svg += f'<text x="{x}" y="{y}" text-anchor="middle" style="font-family:sans-serif;font-size:16pt;" fill="#000">{text}</text>'
         svg += '</svg>'
         return svg
-    
+
     def randomize(self):
         self.display = random.choice(list(self.DISPLAY_WORDS.keys()))
         self.buttons = random.sample(random.choice(self.BUTTON_GROUPS), 6)
         self.log(f"State randomized. Stage {self.stage}. Display: {self.display}. Buttons: {' '.join(self.buttons)}")
-    
+
+    def button_replace_values(self, v):
```

Comment:
As I've said before, add a comment explaining why this function is a no-op. I understand why this is the case, but it doesn't mean I'll remember that forever. Also, I feel like `button_replace_values` isn't really descriptive. My first guess at what this does is "choose new values for the buttons", like what `randomize` currently does. How about calling this `preprocess_input`? I don't think that can be misunderstood.

Reply:
this is for an outdated commit